2022/_data/sponsors.yml

```diff
@@ -114,3 +114,9 @@ supporters:
     image:
       src: "/assets/img/sponsors/jetbrains.png"
       alt: JetBrains
+
+  - name: CodelyTV
+    href: https://codely.tv
+    image:
+      src: "/assets/img/sponsors/codely.png"
```

Comment:
can we try to remove its transparent background ?
you can try for example: https://www.remove.bg/es or https://www.adobe.com/express/feature/image/remove-background

Reply:
Sure! I did not know it was a bad thing. Cheers